docs/README.md

```diff
@@ -44,10 +45,18 @@ return stylus(str)
 }
 ```
 
-_:pushpin: NOTE_: Cozy UI uses [normalize.css](https://necolas.github.io/normalize.css/) by _Nicolas Gallagher_ as a reset styles process. The CSS file is embedded directly by the Cozy UI Stylus plugin, using the stylus `include css` statement to inline the CSS in the output instead of using a native CSS `@import`.
+_📌 NOTE_: Cozy UI uses [normalize.css](https://necolas.github.io/normalize.css/) by _Nicolas Gallagher_ as a reset styles process. The CSS file is embedded directly by the Cozy UI Stylus plugin, using the stylus `include css` statement to inline the CSS in the output instead of using a native CSS `@import`.
```

Comment:
I don't know why this information is in the README, it is an implementation detail to me.

Reply:
Well sure it's a detail, I guess the purpose was to avoid double import in case the developer add it himself, but still, a detail.